sklearn/datasets/base.py

```diff
@@ -352,8 +352,9 @@ def load_iris(return_X_y=False):
         Dictionary-like object, the interesting attributes are:
         'data', the data to learn, 'target', the classification labels,
         'target_names', the meaning of the labels, 'feature_names', the
-        meaning of the features, and 'DESCR', the
-        full description of the dataset.
+        meaning of the features, 'DESCR', the
+        full description of the dataset, 'filename' (added in version 0.19),
```

Comment:
I would put the 0.20 (because 0.19 has been released) below in a separate ``..versionchanged:`` section. That's easier to keep track of if someone is looking for changes, and usually it's irrelevant.